### ISSUE TITLE
Error handling slides (3) shows `'static` lifetime and lifetimes have not been talked about

### ISSUE THREAD

**Issue by amanjeev** [ferrous-systems/rust-training#57]
## Where

Slide 3 of Error handling

## What

Error handling slides (3) shows `'static` lifetime and lifetimes have not been talked about yet in the course.  It makes it harder to introduce this?  Which is basically "ignore this".  Maybe it is ok but still worth this ticket.

## Fix

Maybe use something that does not have a lifetime.

**Comment by jonathanpallant** [comment]
Yeah but at some point they're going to try and use string literals, and they need to know it's painful.

We covered this with an apology and said "don't worry about the details, this is unfortunately how you have to specify *string literal* in Rust as a type"

**Comment by jonathanpallant** [comment]
Dupe of #53?
